packages/components/src/spinner/docs/Spinner.stories.mdx

```diff
@@ -0,0 +1,98 @@
+import { ArgsTable, Meta, Story } from "@storybook/addon-docs";
+import { ComponentInfo, Preview, Tagline } from "@stories/components";
+import { Inline } from "@components/layout";
+import { InnerSpinner, Spinner } from "@components/spinner";
+
+<Meta
+    title="Components/Spinner"
+    id="spinner"
+/>
+
+# Spinner
+
+<Tagline>A spinner indicates that a part of the product is currently loading or performing a task, and the duration of this process is unknown.</Tagline>
+
+<ComponentInfo
+    usage={"import { Spinner } from \"@sharegate/orbit-ui\";"}
+    githubPath="/packages/components/src/spinner/src"
+/>
+
+## Guidelines
+
+### When to use
+
+- To notify users that their request has been received and the action will soon complete.
+
+### Accessibility
+
+- The spinner has a [role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles) of `status` by default. When inside a parent component that is focusable, the spinner should have a role of `presentation` to prevent the screen reader from reading the spinner.
+
+### Spinner vs Loader
+
+- Use Spinner inside a component. When the whole page is waiting for data, use a [loader](?path=/docs/loader--default-story) instead.
+
+## Usage
+
+### Default
+
+A simple spinner.
+
+<Preview>
+    <Story name="default">
+        <Spinner aria-label="Loading..." />
+    </Story>
+</Preview>
+
+### Size
+
+A spinner can vary in size.
+
+<Preview>
+    <Story name="size">
+        <Inline alignY="end">
+            <Spinner size="sm" aria-label="Loading..."/>
+            <Spinner size="md" aria-label="Loading..."/>
+            <Spinner size="lg" aria-label="Loading..."/>
+        </Inline>
+    </Story>
+</Preview>
+
+### Label
+
+A spinner can have a label.
+
+<Preview>
+    <Story name="label">
+        <Inline alignY="end">
+            <Spinner size="sm">Loading...</Spinner>
+            <Spinner size="md">Loading...</Spinner>
+            <Spinner size="lg">Loading...</Spinner>
+        </Inline>
+    </Story>
+</Preview>
+
+
+### Over Background
+
+You can change a spinner style when over a background by setting a color property on the Spinner.
+
+<Preview>
+    <Story name="over background">
+        <Inline alignY="end" backgroundColor="alias-accent" padding={4}>
```

Comment:
Inline is not the right component here, you probably want Flex as you are not horizontally inlining multiple components

Reply:
In fact a regular Div might even be enough for this example